curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804d4.md

```diff
@@ -42,7 +42,7 @@ The `figcaption`'s text should be `Cats hate other cats.` Check for typos and th
 const secondFigCaption = document.querySelectorAll('figcaption')[1];
 assert(
   secondFigCaption &&
-    secondFigCaption.innerText
+    secondFigCaption.innerText.trim()
       .replace(/\s+/gi, ' ')
       .trim()
```

Comment:
this is one of the files that need to be remoded, but here `trim()` is also already present

you should add `trim()` only id it is not already present